cmd/network/verification.go

```diff
@@ -176,13 +178,22 @@ func (e *EgressVerification) Run(ctx context.Context) {
 		out := onv.ValidateEgress(c, *inputs[i])
 		out.Summary(e.Debug)
 
-		// Only suggest sending a service log if the failures are egress-url related
+		// Move cluster into LS if in case cluster can't connect with PD(OR)DMS || service log if the failures other egress-url related
 		if !out.IsSuccessful() && len(out.GetEgressURLFailures()) > 0 {
 			postCmd := generateServiceLog(out, e.ClusterId)
-			if err := postCmd.Run(); err != nil {
+
+			bUrl := strings.Join(postCmd.TemplateParams, ",")
+
+			if strings.Contains(bUrl, "deadmanssnitch") || strings.Contains(bUrl, "pagerduty") {
+				fmt.Println("This cluster traffic has block pagerdury (OR) DMS outgoing traffic")
+				err := putLimitedSupport(e.ClusterId)
```

Comment:
**Question:** do we not have a LS command similarly to `postCmd.Run()` for service logs that we can re-use to avoid code duplication? We would specifically need this to avoid putting "critical customers" into limited support.

Reply:
The postCmd is the o/p of `generateServiceLog` func. it doesn't have LS func

Reply:
Added func in putLimitedSupport to check the critical cu label is present or not

Reply:
I meant that we can follow the same path for posting limited support as we're already following for sending service logs. We're not re-writing the code to send service logs, instead we're re-using what is in the `osdctl servicelog post` command. We can do the same for limited support. [This](https://github.com/openshift/osdctl/blob/master/cmd/cluster/support/post.go) code already posts limited support, we can make it re-usable instead of duplicating code :)

Reply:
I'd exported SendLimitedSupportPostRequest & PrintLimitedSupportReason  func in post.go, In order to re-use here

Reply:
reused post's struct to set cluster into LS. Removed all re-used code